### ISSUE TITLE
Hard coded IP address

### ISSUE THREAD

**Issue by bartonski** [digibib/kohadevbox#105]
The hard coded IP address is inflexible: can't run two devboxes on the same machine without editing the vagrantfile of one. 

At the very least, we could replace

    config.vm.network "private_network", ip: "192.168.50.11"

with

    config.vm.network "private_network", type: "dhcp"

which would take care of the hard coded IP address itself. Ideally, I'd like the option of using a bridged network adapter and making the devbox accessible from my local network.

**Comment by boutros** [comment]
I find the hardcoded IP quite nice, as I can bookmark it, and don't have to look for it everytime i boot the devbox.

**Comment by bartonski** [comment]
boutros,

If the network stays set as `"private_network"`, the IP address will be assigned by DHCP hosted on your local computer. This will, in all likelihood, stay the same every time you bring up kohadevbox. The first time you log into kohadevbox using `vagrant ssh`, you should be able to find the ip address by running `ip addr`. You can then bookmark that address.

**Comment by boutros** [comment]
Yes, It's easy enough to find, just a minor inconvenience. It's no big deal for me. Go ahead :)
